virttest/utils_netperf.py

```diff
@@ -124,8 +124,13 @@ def pack_compile(self, compile_option=""):
         build_type = {"aarch64": "aarch64-unknown-linux-gnu"}
         build_arch = self.session.cmd_output("arch", timeout=60).strip()
         np_build = build_type.get(build_arch, build_arch).strip()
-        setup_cmd = "./configure --build=%s %s > /dev/null 2>&1" % (np_build,
-                                                                    compile_option)
+        if self.netperf_dir == "/var/tmp/netperf-2.7.1":
```

Comment:
Check the absolute path is not a good idea, better to use RegExp to judge its version from `self.netperf_file` and then add `autogen.sh` before currently command if need.

Reply:
Checking if `autogen.sh` exists is an easier method.

Reply:
I have updated it,please help me review it again.Thanks.